bitnami/redis-cluster/values.yaml

```diff
@@ -744,12 +744,15 @@ cluster:
   ##
   update:
     ## @param cluster.update.addNodes Boolean to specify if you want to add nodes after the upgrade
-    ## Setting this to true a hook will add nodes to the Redis&trade; cluster after the upgrade. currentNumberOfNodes is required
+    ## Setting this to true a hook will add nodes to the Redis&trade; cluster after the upgrade. currentNumberOfNodes and currentNumberOfReplicas is required
```

Comment:
Please update the validation into  `_helpers.tpl` to check the necessary parameters are set in that case

Reply:
Done.Please review it again.Thank you^_^